src/couch/src/couch_httpd_auth.erl

```diff
@@ -28,7 +28,7 @@
 -export([cookie_auth_header/2]).
 -export([handle_session_req/1, handle_session_req/2]).
 
--export([authenticate/4, verify_totp/2]).
```

Comment:
apparently not used externally, so I removed the export rather than update the arity.